.gitignore

```diff
@@ -1,13 +1,18 @@
 /.idea
+/.composer/
```

Comment:
Not sure why this directory popped up... Might be a Composer mismanipulation from my part or the Symfony flex plugin. In any case I don't think it really hurts to have it here

Reply:
Shouldn't it've been in `$HOME`?

Reply:
yeah... I have no idea why really